.gitignore

```diff
@@ -1 +1,3 @@
-TEST-*.xml
+.bundle/
+vendor/
+Gemfile.lock
```

Comment:
The usual question: should Gemfile.lock be ignored?

Reply:
as we learned: we should add it. i will update the pr for this.